taskweaver/memory/attachment.py

```diff
@@ -35,6 +35,9 @@ class AttachmentType(Enum):
     # CodeInterpreter - revise code
     revise_message = "revise_message"
 
+    # function calling
+    function = "function"
+
```

Comment:
Do we need a new Attachment type if the generated function could still be regarded as chunk of code as before?

Reply:
just want to make it clearer